src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
-// write your code here
+const population = document.querySelectorAll('span.population');
+const totalPlace = document.querySelector('span.total-population');
+const averagePlace = document.querySelector('span.average-population');
+
+const numberedPopulation = [...population].map(item => {
+  const number = Number(item.textContent.split(',').join(''));
+
+  return number;
```

Comment:
You don't use the variable
```suggestion
  return Number(item.textContent.split(',').join(''));
```